src/components/backupWallet/ExportWarningPopup.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { View } from 'react-native'
+import { noop } from 'lodash'
+
+import Icon from '../common/view/Icon'
+import ExplanationDialog from '../common/dialogs/ExplanationDialog'
+
+import { withStyles } from '../../lib/styles'
+
+import { getDesignRelativeHeight, getDesignRelativeWidth } from '../../lib/utils/sizes'
+
+const mapImageStylesToProps = ({ theme }) => ({
+  imageWrapper: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  image: {
+    color: theme.colors.primary,
+    transform: 'rotate(180deg)',
```

Comment:
```suggestion

```
this silently crashes because of applying transform to text. instead it could be applied to imageWrapper and the result is the same but without the error